cmd/flow-dps-live/main.go

```diff
@@ -91,6 +92,7 @@ func run() int {
 	pflag.StringVarP(&flagLevel, "level", "l", "info", "log output level")
 	pflag.StringVar(&flagSeedAddress, "seed-address", "", "address of the seed node to follow unstaked consensus")
 	pflag.StringVar(&flagSeedKey, "seed-key", "", "hex-encoded public network key of the seed node to follow unstaked consensus")
+	pflag.BoolVar(&flagSkip, "skip-registers", false, "skip indexing of execution state ledger registers")
```

Comment:
Just as a fun fact, since you mentioned this is intended for testing, `pflag` library has an option to make flags hidden.

https://pkg.go.dev/github.com/spf13/pflag#FlagSet.MarkHidden